config/installbundle/releases/scopes/namespaced/per-namespace-components/kustomization.yaml

```diff
@@ -19,7 +19,7 @@ commonLabels:
 commonAnnotations:
   cnrm.cloud.google.com/version: 0.0.0-dev
 nameSuffix:
-  -mynamespace
+  - mynamespace
```

Comment:
The original `-mynamespace` is correct. ( Other fixes are really impressive!)

Reply:
Oh gosh, the suffix is actually `-mynamespace`.

Is it possible to change it to `"-mynamespace"` to make that more clear?

Anyway, I'll remove this since any such change wouldn't really fit with this PR.